BlockEntityBehavior/BEBehaviorTrapDoor.cs

```diff
@@ -80,6 +97,7 @@ private Matrixf getTfMatrix(float rotz=0)
                 return new Matrixf()
                     .Translate(0.5f, 0.5f, 0.5f)
                     .RotateYDeg(RotDeg)
+                    .RotateZDeg(BlockFacing.ALLFACES[AttachedFace].Negative ? 180 : 0)

```

Comment:
This flips it so you can place trapdoors against the bottom of blocks like you could in 1.19.